scripts/build.js

```diff
@@ -11,8 +11,8 @@ methodFiles.forEach(file => {
 	let parsed = JSON.parse(raw);
 	methods = [
 		...methods,
-		...parsed,
-	];
+	  ...parsed
```

Comment:
I think this change was a mistake?  Most of this file is using tab for indentation, but some lines are using spaces.  This should be fixed file/repository wide, but at least for this PR we probably shouldn't be making the situation worse.  😄 

I recommend adding a `.editorconfig` to the root of the project so people's editors can automatically do the right thing with regards to indentation consistency.

Reply:
I'll add an `.editorconfig`. I feel relatively strongly that this should be a tab indented repo. There is very little alignment occurring and I think people should have the choice of how the view the specs. Of course, if there are good reasons against, I can be convinced otherwise....

Reply:
Okay, walking this back slightly. I think I'm okay for spaces in the js files as it appears to be the standard. We'll keep the spec files tab indented.

Reply:
Boo.  Tabs are definitely superior here for the exact reason you mentioned.